src/requests/ajax.js

```diff
@@ -3,15 +3,15 @@ import axios from 'axios';
 import { DOMAIN, PATH } from '../utils/constants';
 
 export async function loginRequest(name, passcode) {
-  try {
-    // login and return the user and token
-    const res = await axios.post(`${DOMAIN}${PATH.users}/chat`, {
-      name,
-      passcode,
-    });
-    return res.data; // {user, token}
-  } catch (error) {
-    // return the error message
-    return error.response.data.message;
-  }
+  // login and return the user and token
+  const res = await axios.post(`${DOMAIN}${PATH.users}/chat`, {
+    name,
+    passcode,
+  });
+  return res.data; // {user, token}
+}
+
+export async function getHotel(hotel_id) {
+  const res = await axios.get(`${DOMAIN}${PATH.hotel}/${hotel_id}`);
+  return res.data;
```

Comment:
Why remove the `try-catch` blocks? This can cause unhandled promise rejections

Reply:
It is handled in the component if I handle here and return it will never hit the catch in the component